src/documentation/documentationengine.cpp

```diff
@@ -638,22 +633,34 @@ void DocumentationEngine::writeDocumentation() const {
     sceneProperties[NameKey] = SettingsTitle;
     sceneGraph[NameKey] = SceneTitle;
 
-    // Add this here so that the generateJson function is the same as before to ensure
-    // backwards compatibility
-    nlohmann::json scriptingResult;
-    scriptingResult[NameKey] = ScriptingTitle;
-    scriptingResult[DataKey] = scripting;
-
     nlohmann::json documentation = {
-        sceneGraph, sceneProperties, actions, events, keybindings, license,
-        scriptingResult, factory
+        sceneGraph, sceneProperties, actions, events, keybindings, license
     };
 
     nlohmann::json result;
     result[DocumentationKey] = documentation;
 
+    // Make into a javascript variable so that it is possible to open with static html
     std::ofstream out = std::ofstream(absPath("${DOCUMENTATION}/documentationData.js"));
     out << "var data = " << result.dump();
+    out.close();
```

Comment:
```suggestion
```

No need to explicitly close as the closing scope will do that automatically

Reply:
When I didn't do it it didn't create a new file but printed to the same file again.